src/formElementPlugins/shortText/src/ApplicationView.tsx

```diff
@@ -30,9 +30,9 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
 
   return (
     <>
+      <label>{label}</label>
       <Form.Input
```

Comment:
Oh, okay, so you can leave it as `<Form.Input>` rather than `<Input>`. Do you know what the difference is?

Reply:
I think the main difference is that with the Input we have to change how we do the validation - it expects only a `boolean` for the error and then another label on the bottom with the `validationMessage`. I have changed to be like this, and then changed back to use Form.Input, so there was something else. I'll check what it was.

Reply:
Here is the difference:

With `<Input>`

<img width="492" alt="Screen Shot 2020-12-02 at 4 48 16 PM" src="https://user-images.githubusercontent.com/16461988/100826033-5639ef00-34be-11eb-99f6-87d525f9b9c1.png">

With `<Form.Input>`

<img width="500" alt="Screen Shot 2020-12-02 at 4 43 52 PM" src="https://user-images.githubusercontent.com/16461988/100826039-5934df80-34be-11eb-88f1-850d72a4db13.png">

Reply:
OH, okay. That's weird, because we were already using `<Form.Input>` and NOT getting the red borders, right? 🤷‍♂️

But looks good 👍